api_tests/me_patch_others.yaml

```diff
@@ -0,0 +1,64 @@
+# Test generated by RoostGPT for test credential-wallet-api using AI Type Open AI and AI Model gpt-4
+
+
+# RoostTestHash=6dd436357d
+
+config:
+  target: "{{ $processEnvironment.API_HOST }}"
+  plugins:
+    expect: {}
+  payload:
+    path: "me_patch_others.csv"
+    skipHeader: true
+    fields:
+      - "name"
+      - "image"
+      - "settings"
+  phases:
+    - duration: 60
+      arrivalRate: 5
+scenarios:
+  - name: "Non-Success API Responses"
+    flow:
+      - log: "Testing with invalid payload"
+        patch:
+          url: "/me"
+          json:
+            name: "{{ name }}"
+            image: "{{ image }}"
```

Comment:
Type of the image could have been picked better (base64 rather than string) for CSV generation
Prefer to use boty : type + format  rather than just type